### DISCUSSION TITLE
Create collectionoffer on Polygon chain using Sdk

### DISCUSSION THREAD

**Discussion by wainesiro** [ProjectOpenSea/opensea-js#1397] [Q&A]
Hello, I would like to know how do i create Collectionoffers using Opensea Sdk. i tried to change the "Chain" and "Paymenttokenaddress" but keep getting errors

**Comment by ryanio** [comment]
See the quick start for selecting the chain: https://github.com/ProjectOpenSea/opensea-js/blob/main/developerDocs/quick-start.md

And use method `sdk.createCollectionOffer()`. What error(s) are you experiencing? If you use TypeScript you should get helpful errors if you are using the method wrong or passing incorrect parameters.
